Testscripts/Windows/VERIFY-DPDK-OVS.ps1

```diff
@@ -60,6 +63,26 @@ function Main {
 		Write-LogInfo "  SSH Port : $($serverVMData.SSHPort)"
 		Write-LogInfo "  Internal IP : $($serverVMData.InternalIP)"
 
+		# Checking OVS DPDK compatibility
+		$detectedDistro = Detect-LinuxDistro -VIP $vmData.PublicIP -SSHport $vmData.SSHPort `
```

Comment:
use the global detectedDistro as in here - https://github.com/LIS/LISAv2/blob/master/Testscripts/Windows/GPU-DRIVER-INSTALL.ps1#L136